codegen/test/chart-no-desc/values.schema.json

```diff
@@ -8076,6 +8194,67 @@
                                   "type": "null"
                                 }
                               ]
+                            },
+                            "schedulingGates": {
```

Comment:
looks like these and other schema.json updates came from new fields being added to the pod spec in the latest k8s api